db/schema.rb

```diff
@@ -40,20 +27,22 @@
     t.integer "likes_counter"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "author_id", null: false
-    t.index ["author_id"], name: "index_posts_on_author_id"
+    t.bigint "user_id", null: false
```

Comment:
- [ ] I suggest we go back to `author_id` because that is how the ERD shows, see the image below. We can also change the model associations to match the design. For example, `belongs_to :author, class_name: 'User'`

![authorid](https://user-images.githubusercontent.com/53124548/160345607-0748b101-4e2d-40ff-a52b-240c2fefff10.png)